examples/regional-dlp/main.tf

```diff
@@ -22,6 +22,7 @@ locals {
 module "data_ingestion" {
   source                           = "../.."
   org_id                           = var.org_id
+  labels                           = var.labels
```

Comment:
I thought in an example we'd actually hard code something here like "environment:dev"  or "purpose:ingestion";  otherwise, can we just eliminate it since the main module defaults to none.

If we do add a label for each example, let's make it consistent for every example.  I don't think we'll need a variables file update in the examples.